common/v2/services/LocalCache/LocalCache.ts

```diff
@@ -177,10 +179,11 @@ export const initDerivationPathOptions = () => {
   });
   setCache(newStorage);
 };
+*/
 
 // Low level operations
 
-const getCacheRaw = (): LocalCache => {
+export const getCacheRaw = (): LocalCache => {
```

Comment:
getCacheRaw probably shouldn't be used outside of this file since it has no error handling and doesn't ensure that the cache fields exist in advance.

```suggestion
const getCacheRaw = (): LocalCache => {
```